packages/nextjs/utils/scaffold-eth/getMetadata.ts

```diff
@@ -0,0 +1,30 @@
+export const getMetadata = ({
+  title,
+  description,
+  imageRelativePath = "/thumbnail.jpg",
+}: {
+  title: string;
+  description: string;
+  imageRelativePath?: string;
+}) => {
```

Comment:
Lets do `}): Metadata => {`

So that people don't need to do  `export const metadata: Metadata` in every page they use `getMetadata` function and it will inferred instead

Reply:
done 👍